magit-gh-comments-github.el

```diff
@@ -31,11 +33,27 @@
 (defun magit-gh--url-for-pr-comments (pr)
   (format "%s/comments" (magit-gh--url-for-pr pr)))
 
-(cl-defun magit-gh--request-sync (url &rest request-args
-                                      &key
-                                      (parser #'buffer-string)
-                                      (timeout magit-gh--request-timeout-seconds)
-                                      &allow-other-keys)
+(defun magit-gh--url-for-pr-reviews (pr)
+  (format "%s/reviews" (magit-gh--url-for-pr pr)))
+
+;; TODO: Put this back, just don't want to wipe out my cache without wifi
+(setq magit-gh--request-cache nil)
+(cl-defun magit-gh--request-sync (url &rest request-args)
```

Comment:
Revisit whether we actually need this. If we do, can we use HEAD requests in combination with etag? Or add a TTL? As is, this is preventing us from redisplaying comments.